src/Compilers/CSharp/Test/Syntax/Parsing/FuzzTesting.cs

```diff
@@ -81,18 +81,25 @@ public void c024928()
 
         [WorkItem(2771, "https://github.com/dotnet/roslyn/issues/2771")]
         [Fact]
-        public void TestBinary()
+        public unsafe void TestBinary()
         {
             // use a fixed seed so the test is reproducible
             var random = new System.Random(12345);
```

Comment:
As an aside, how is this fuzz testing? we pass the same seed to the pseudo-random generator each time.  Why not use a better random number generator and report the seed in the event of test failure so a dev could reproduce it?